### PR TITLE
Updated artifacts page for image-builder

### PR DESCRIPTION
**Pull request by chrisnegus**
*Description of changes:*
This PR updates the artifacts.md page to tell users that:
- EKS Anywhere no longer distributes Ubuntu images and
- They can use image-builder to create their own Ubuntu-based node images

### CODE REVIEW

File: docs/content/en/docs/reference/artifacts.md [hunk 1]
```diff
@@ -97,125 +85,155 @@ EKS-D Release
...
+
+* Run on Ubuntu 22.04 or later 
+* Machine requirements:
+  * AMD 64-bit architecture
+  * 50 GB disk space
+  * 2 vCPUs
+  * 8 GB RAM
+  * Bare Metal only: Run on a bare metal machine with virtualization enabled
+* Network access to:
+  * vCenter endpoint
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply docs/content/en/docs/reference/artifacts.md:127]: Done.

File: docs/content/en/docs/reference/artifacts.md [hunk 2]
```diff
@@ -97,125 +85,155 @@ EKS-D Release
...
+  * public.ecr.aws (to download container images from EKS Anywhere)
+  * anywhere-assets.eks.amazonaws.com (to download the EKS Anywhere binaries, manifests and OVAs)
+  * distro.eks.amazonaws.com (to download EKS Distro binaries and manifests)
+  * d2glxqk2uabbnd.cloudfront.net (for EKS Anywhere and EKS Distro ECR container images)
+* Optional proxy configuration: You can use a proxy server to route outbound requests to the internet. To configure `image-builder` tool to use a proxy server, export these proxy environment variables:
+  ```
+  export HTTP_PROXY=<HTTP proxy URL e.g. http://proxy.corp.com:80>
+  export HTTPS_PROXY=<HTTPS proxy URL e.g. http://proxy.corp.com:443>
+  export NO_PROXY=<No proxy>
+  ```
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply docs/content/en/docs/reference/artifacts.md:137]: I added it as its own section.

File: docs/content/en/docs/reference/artifacts.md [hunk 3]
```diff
@@ -97,125 +85,155 @@ EKS-D Release
...
+1. Install packages and prepare environment:
+   ```
+   sudo apt update -y
+   sudo apt install jq unzip make ansible -y
+   sudo snap install yq
+   ```
+1. Get `image-builder`:
+   ```bash
+   cd /tmp
+   sudo wget https://projectbuildpipeline-857-pipelineoutputartifactsb-10ajmk30khe3f.s3.us-west-2.amazonaws.com/projects/aws/image-builder/latest/image-builder-linux-amd64-0.1.0.tar.gz
```
[the remark this answers is not in the document]
Comment (chrisnegus) [reply docs/content/en/docs/reference/artifacts.md:179]: Okay. I'll remove it for now and put a placeholder.

**Review by vignesh-goutham — APPROVED** [review]
/approve

File: docs/content/en/docs/reference/artifacts.md [hunk 4]
```diff
@@ -6,30 +6,26 @@ description: >
...
-Kubernetes 1.21:
-```bash
-https://anywhere-assets.eks.amazonaws.com/releases/bundles/12/artifacts/raw/1-21/ubuntu-v1.21.13-eks-d-1-21-16-eks-a-12-amd64.gz
-```
-
-Kubernetes 1.22:
-```bash
-https://anywhere-assets.eks.amazonaws.com/releases/bundles/12/artifacts/raw/1-22/ubuntu-v1.22.10-eks-d-1-22-9-eks-a-12-amd64.gz
-```
+EKS Anywhere no long distributes Ubuntu OS images.
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/artifacts.md:25]: "no longer"

File: docs/content/en/docs/reference/artifacts.md [hunk 5]
```diff
@@ -99,131 +95,172 @@ EKS-D Release
...
-8. Update the Ubuntu configuration file with the new custom ISO URL and its checksum at
-`image-builder/images/capi/packer/ova/ubuntu-2004.json`
-9. Setup image-builder and run the OVA build for the Kubernetes version.
-```
-RELEASE_BRANCH=1-23 make release-ova-ubuntu-2004
-```
-
-# Images
+### Ubuntu OVAs
+EKS Anywhere no longer distributes Ubuntu OVAs for use with EKS Anywhere clusters.
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/artifacts.md:99]: Can we make this sentence consistent with above?

File: docs/content/en/docs/reference/artifacts.md [hunk 6]
```diff
@@ -99,131 +95,172 @@ EKS-D Release
...
+   ├── hook
+   │   ├── initramfs-x86_64
+   │   └── vmlinuz-x86_64
+   └── my-ubuntu-v1.22.10-eks-d-1-22-8-eks-a-11-amd64.gz
+   ```
+
+1. Then create the [Bare metal configuration]({{< relref "./clusterspec/baremetal/" >}}) file, setting the `osImageURL` field to the location of the image. For example:
+
+   ```
+   osImageURL: "http://my-web-server/my-ubuntu-v1.22.10-eks-d-1-22-8-eks-a-11-amd64.gz"
```
Comment (vivek-koppuru) [inline docs/content/en/docs/reference/artifacts.md:258]: Update this to reference 1.23

**Pull request merged**